client/my-sites/upgrades/checkout/credits-payment-box.jsx

```diff
@@ -46,14 +46,9 @@ var CreditsPaymentBox = React.createClass( {
 	},
 
 	render: function() {
-		var classSet = classNames( {
-			'credits-payment-box': true,
-			selected: this.props.selected === true
-		} );
-
 		return (
 			<PaymentBox
-				classSet={ classSet }
+				classSet="credits-payment-box"
```

Comment:
This is fine, but in the future let's stick to more targeted PRs. It's very easy to regress in behavior when we try to fix several things at once.